fiona/fio/load.py

```diff
@@ -23,6 +24,7 @@
 @options.src_crs_opt
 @click.option('--dst-crs', '--dst_crs',
               help="Destination CRS.  Defaults to --src-crs when not given.")
+@cligj.features_in_arg
 @click.option('--sequence / --no-sequence', default=False,
               help="Specify whether the input stream is a LF-delimited "
                    "sequence of GeoJSON features (the default) or a single "
```

Comment:
@sid-kap the sequence option and and argument for `load()` can be deleted. `cligj.features_in_arg` figures out whether the input is a sequence or not with no option needed.